.changeset/stupid-lions-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-error-view': minor
+---
+
+Uses the useErrorPageAnalytics hook
```

Comment:
Doesn't look like we are leveraging the `useErrorPageAnalytics` hook on this implementation.

```suggestion
Replaces track event name with static value.
```

Reply:
Oops!